apps/prairielearn/src/batched-migrations/20250501105324_ai_grading_job_prompt_backfill.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+
+import { makeBatchedMigration } from '@prairielearn/migrations';
+import { loadSqlEquiv, queryAsync, queryRow } from '@prairielearn/postgres';
+
+const sql = loadSqlEquiv(import.meta.url);
+
+export default makeBatchedMigration({
+  async getParameters() {
+    // Should backfill all existing rows of ai_grading_jobs
+    const { min, max } = await queryRow(
+      sql.select_bounds,
+      z.object({
+        min: z.bigint({ coerce: true }).nullable(),
+        max: z.bigint({ coerce: true }).nullable(),
+      }),
+    );
+    return { min, max, batchSize: 10_000 };
```

Comment:
Nit (unscientific), let's lower this a bit.

```suggestion
    return { min, max, batchSize: 1000 };
```

Reply:
Resolved in https://github.com/PrairieLearn/PrairieLearn/pull/11889/commits/eec550816bddef71148e9cc9cbda8b1e8e631feb